helpers.go

```diff
@@ -0,0 +1,26 @@
+package xattr
+
+// Error records an error and the operation, file path and attribute that caused it.
+type Error struct {
+	Op   string
+	Path string
+	Name string
+	Err  error
+}
+
+func (e *Error) Error() string {
+	return e.Op + " " + e.Path + " " + e.Name + ": " + e.Err.Error()
+}
+
+// nullTermToStrings converts an array of NULL terminated UTF-8 strings to a
+// []string. Used on Darwin and Linux.
+func nullTermToStrings(buf []byte) (result []string) {
```

Comment:
`xattr.go` contains all common stuff. I do not see any reason to add another, generic file like `helpers` with `Error` definition and private function. Lets keep it in `xattr.go`

Reply:
Ok